modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalNavTag.java

```diff
@@ -294,6 +301,28 @@ private void _renderVerticalNavItems(
 			jspWriter.write(
 				HtmlUtil.escape((String)verticalNavItem.get("label")));
 
+			List<IconItem> iconItems = (List<IconItem>)verticalNavItem.get(
+				"icons");
+
+			if (ListUtil.isNotEmpty(iconItems)) {
+				for (IconItem iconItem : iconItems) {
+					String symbol = (String)iconItem.get("symbol");
+
+					if (Validator.isNotNull(symbol)) {
+						_writeIcon(symbol, jspWriter);
+					}
+				}
+			}
+
+			if (GetterUtil.getBoolean(verticalNavItem.get("deprecated"))) {
```

Comment:
I don't see a reason why we couldn't use tag from another module. If the inner tag has any attributes required for rendering, like the namespace, it can be set in a setter, like `iconTag.setSymbol("caret-bottom");` in this file.

Reply:
We cannot use the taglib, because this taglib is in frontend-taglib adding the dependency to this module cause a circular dependency

Reply:
I mean using `FeatureIndicatorTag.java`